pkg/operator/controller/dns_status_test.go

```diff
@@ -156,7 +176,18 @@ func TestDNSStatusConditions(t *testing.T) {
 		if !tc.inputs.haveClusterIP {
 			haveClusterIP = "no cluster ip"
 		}
-		description := fmt.Sprintf("%s, %d/%d DNS pods available, %d/%d node-resolver pods available", haveClusterIP, tc.inputs.availDNS, tc.inputs.desireDNS, tc.inputs.availNR, tc.inputs.desireNR)
+		var managementState string
+		switch tc.inputs.managementState {
+		case operatorv1.Managed:
+			managementState = "Managed"
+		case operatorv1.Force:
+			managementState = "Force"
+		case operatorv1.Removed:
+			managementState = "Removed"
+		case operatorv1.Unmanaged:
+			managementState = "Unmanaged"
+		}
+		description := fmt.Sprintf("%s, %d/%d DNS pods available, %d/%d node-resolver pods available, managementState is %s", haveClusterIP, tc.inputs.availDNS, tc.inputs.desireDNS, tc.inputs.availNR, tc.inputs.desireNR, managementState)
```

Comment:
Can you use `tc.inputs.managementState` directly in the print statement?
```suggestion
		description := fmt.Sprintf("%s, %d/%d DNS pods available, %d/%d node-resolver pods available, managementState is %s", haveClusterIP, tc.inputs.availDNS, tc.inputs.desireDNS, tc.inputs.availNR, tc.inputs.desireNR, string(tc.inputs.managementState))
```
We can merge as-is leave that for a follow-up though.